core/hybrisadaptor.cpp

```diff
@@ -882,10 +892,27 @@ void HybrisManager::startReader(HybrisAdaptor *adaptor)
 void HybrisManager::stopReader(HybrisAdaptor *adaptor)
 {
     if (m_registeredAdaptors.values().contains(adaptor)) {
-            sensordLogD() << "deactivating " << adaptor->name();
-            if (!setActive(adaptor->m_sensorHandle, false)) {
-                sensordLogW() <<Q_FUNC_INFO<< "failed";
-            }
+        sensordLogD() << "deactivating " << adaptor->name();
+        if (!setActive(adaptor->m_sensorHandle, false)) {
+            sensordLogW() <<Q_FUNC_INFO<< "failed";
```

Comment:
Old code but could have spaces around Q_FUNC_INFO.

As side note, these logging functions seem like pointless wrappers for qt. Could switch to qt category logging, but that's for different PR.

Reply:
spaces added